docs/tutorials/preload.md

```diff
@@ -0,0 +1,55 @@
+# Preload API
+
+In some situations, you may wish to begin loading an asset before Shaka Player
+has been attached to a video, or before you have fully committed to playing it.
+Perhaps you have a menu in which users select multiple assets, and you want to
+preload the assets they mouse over to reduce load latency when those assets are
+selected.
+In situations like that, the Shaka Player preload API can be useful.
+
+
+#### Basic Usage
+
+To preload an asset, call the preload method on a player instance.
+
+```js
+async function initPlayer() {
+  const player = new shaka.Player();
+  const preloadManager = await player.preload(
+    'https://storage.googleapis.com/shaka-demo-assets/angel-one/dash.mpd');
+  if (preloadManager) {
+    // The asset is something that can be preloaded. Once you have this manager,
+    // you can load it later by passing it to the load method:
+    await player.load(preloadManager);
+    // If you decide to not play the preloaded asset, you can instead destroy
+    // the preload manager:
+    await preloadManager.destroy();
+  } else {
+    // This asset is something that cannot be preloaded (for instance, a raw
```

Comment:
It should mention that there may be no preloadManager in case of HLS-Safari

Reply:
Done.